internal/wasm/jit/asm/amd64/consts.go

```diff
@@ -0,0 +1,206 @@
+package amd64
+
+import "github.com/tetratelabs/wazero/internal/wasm/jit/asm"
+
+// Arm64-specific register states.
+const (
+	ConditionalRegisterStateE  = asm.ConditionalRegisterStateUnset + 1 + iota // ZF equal to zero
+	ConditionalRegisterStateNE                                                //˜ZF not equal to zero
+	ConditionalRegisterStateS                                                 // SF negative
+	ConditionalRegisterStateNS                                                // ˜SF non-negative
+	ConditionalRegisterStateG                                                 // ˜(SF xor OF) & ˜ ZF greater (signed >)
+	ConditionalRegisterStateGE                                                // ˜(SF xor OF) greater or equal (signed >=)
+	ConditionalRegisterStateL                                                 // SF xor OF less (signed <)
+	ConditionalRegisterStateLE                                                // (SF xor OF) | ZF less or equal (signed <=)
+	ConditionalRegisterStateA                                                 // ˜CF & ˜ZF above (unsigned >)
+	ConditionalRegisterStateAE                                                // ˜CF above or equal (unsigned >=)
+	ConditionalRegisterStateB                                                 // CF below (unsigned <)
+	ConditionalRegisterStateBE                                                // CF | ZF below or equal (unsigned <=)
+)
+
+// Arm64-specific instructions.
+const (
```

Comment:
this is interesting list as this is the list of all instructions used in wazero!